views/auth/booInfo.ejs

```diff
@@ -0,0 +1,9 @@
+<form action="/auth/booInfo.ejs" method="POST">
+    <label for="booName">Boo Name</label>
+    <input id="booName" type="text" name="name">
+
+    <label for="booNumber">Boo Number</label>
+    <input id="number" type="number" name="number">
```

Comment:
The id should match the `for` tag in the `label` element. I say change this id to booNumber!